### PR TITLE
Reduce size of stream analysis docker image

### PR DESCRIPTION
**Pull request by bstrzele**

### CODE REVIEW

File: demos/real_time_stream_analysis/python/install_opencv.sh [hunk 1]
```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```
Comment (dkalinowski) [inline demos/real_time_stream_analysis/python/install_opencv.sh:1]: This new file duplicates model_server/third_party/opencv/install_opencv.sh

File: demos/real_time_stream_analysis/python/Dockerfile [hunk 2]
```diff
@@ -16,7 +16,9 @@
 
 FROM ubuntu:20.04
 ENV DEBIAN_FRONTEND="noninteractive"
-RUN apt -y update && apt install -y libopencv-dev python3-opencv python3-pip
+RUN apt -y update && apt install -y --no-install-recommends libgl1 libglib2.0-0 python3-pip
+COPY install_opencv.sh /real_time_stream_analysis/
+RUN bash /real_time_stream_analysis/install_opencv.sh
 COPY requirements.txt /real_time_stream_analysis/
 RUN pip3 install -r /real_time_stream_analysis/requirements.txt
```
Comment (dtrawins) [inline demos/real_time_stream_analysis/python/Dockerfile:23]: add --no-cache-dir

File: demos/real_time_stream_analysis/python/Dockerfile [hunk 3]
```diff
@@ -16,7 +16,9 @@
 
 FROM ubuntu:20.04
 ENV DEBIAN_FRONTEND="noninteractive"
-RUN apt -y update && apt install -y libopencv-dev python3-opencv python3-pip
+RUN apt -y update && apt install -y --no-install-recommends libgl1 libglib2.0-0 python3-pip
```
Comment (dtrawins) [inline demos/real_time_stream_analysis/python/Dockerfile:19]: use apt-get instead of apt
add  && rm -rf /var/lib/apt/lists/*

**Review by dtrawins — CHANGES_REQUESTED** [review]

**Review by atobiszei — APPROVED** [review]

**Review by dkalinowski — APPROVED** [review]

**Comment by mzegla** [comment]
What's the final size of the image?

**Comment by bstrzele** [comment]
Final size: 526MB

**Review by mzegla — APPROVED** [review]
Let's separately explore minimal installation possibility to see if we can shrink the image even more.

**Review by dtrawins — APPROVED** [review]

**Pull request merged**